tournamentcontrol/competition/admin.py

```diff
@@ -476,6 +477,7 @@ def get_urls(self):
                 path("<uuid:person_id>/", self.edit_person, name="edit"),
                 path("<uuid:person_id>/delete/", self.delete_person, name="delete"),
                 path("<uuid:person_id>/merge/", self.merge_person, name="merge"),
+                path("<uuid:person_id>/transfer/", self.transfer_person, name="transfer"),
```

Comment:
The transfer link needs to be exposed in the templates to make it accessible from the frontend.